src/psbt.h

```diff
@@ -1423,7 +1426,7 @@ struct PartiallySignedTransaction
             // Scalar offsets
             for (const uint256& scalar : m_scalar_offsets) {
                 SerializeToVector(s, CompactSizeWriter(PSBT_GLOBAL_PROPRIETARY), PSBT_ELEMENTS_ID, CompactSizeWriter(PSBT_ELEMENTS_GLOBAL_SCALAR), scalar);
-                SerializeToVector(s, std::vector<unsigned char>());
+                s << PSBT_SEPARATOR; /* Zero length data value */
```

Comment:
I actually don't understand the difference between this line and its replacement. Don't they both just serialize a 0x00?

Reply:
Oh no, I'm an idiot, the original line serilaizes two `00`s, because it's serializing a vector in a vector. Okay.